### DISCUSSION TITLE
UX

### DISCUSSION THREAD

**Discussion by DarirAbdelbassit** [community/community#73136] [Other Feature Feedback, Questions, & Ideas] [answered]
### It's sucks

Bad user experience

### Body

I did not like the pagination it's too long

**Comment by mhaemnn** [comment]
which pagination do you mean?

**Reply by DarirAbdelbassit** [reply]
All paginations (followers, repositories, etc.) should be limited to a maximum of 10. What is the purpose of pagination if I have to keep scrolling to find the 'Next' button?

**Reply by mhaemnn** [reply]
Can you take photos or videos, who knows, I can help

**Reply by DarirAbdelbassit** [reply]
![pagination-lenght](https://github.com/community/community/assets/85806305/c23bfd1f-0d46-4c2b-8a41-d33517ff120a)

**Comment by Esra404** [comment]
When you notice that all your repos are shown on a single page on GitHub and the option to move forward does not appear when the page scrolls down, there is probably an issue or bug in GitHub's user interface. Here are the steps to solve this type of problem:

Clean and Update the Browser: First, clean the web browser you are using (clear cookies and cache) and update it. It may be due to some browser bugs, updates or cookie issues.

Use a Different Browser: If the issue is browser specific, try using a different web browser. If the problem persists in a different browser, it may not be browser dependent.

Disable Browser Add-ons: Browser add-ons or add-ons can sometimes cause incompatibility issues with websites. Try disabling your browser's add-ons.

Use a Different Device or Network: If possible, try opening GitHub using a different device or network (Wi-Fi or mobile data). This can help you determine if there is an issue with the network or your device.

Contact GitHub Support: If the above steps do not resolve your issue, contacting GitHub support may be the best option. GitHub support team can guide you with further assistance and resolving your issue.

Such an issue on the GitHub user interface is usually a temporary issue and is resolved as a result of fixes on GitHub's systems. However, you can still try to solve your problem by trying the steps above.

**Reply by DarirAbdelbassit** [reply, marked as answer]
No, there is no issue with my device, the problem is that the number chosen by GitHub is too high.

**Comment by mhaemnn** [comment]
I've tried to find out, but I haven't found what I want, but to set it it has to be in the type or indeed Github doesn't set it so it's easier when looking for other repositories
